src/v3/src/img/16pxDevice.svg

```diff
@@ -1,5 +1,5 @@
 <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg" aria-labelledby="desktop" role="img">
-<title id="desktop-icon">Browser</title>
+<title id="desktop-icon">Desktop browser</title>
```

Comment:
Are these considered english leaks?

Reply:
These may be a good candidate for using aria-hidden since they are really just for visual effect. 

cc: @alexdahl-okta

Reply:
I think that's fair! We're using these icons as visual labels (instead of text labels) to decrease cognitive load and increase speed to the content. For low-vision users, they probably also want to get right to the content, and the content should be self-evident and sufficient for users to evaluate if the request is legitimate.

Reply:
yeah looks like we will use aria-hidden on these.  I added them at first to see if the spec test fail, which it did since we cant find by title when it is aria-hidden.  I removed it because I was thinking this should be added in the work for this ticket https://oktainc.atlassian.net/browse/OKTA-594789

Reply:
if its okay to just add it here I can do that